insalan/payment/admin.py

```diff
@@ -0,0 +1,73 @@
+# Disable lints:
+# "Too few public methods"
+# pylint: disable=R0903
+
+from django import forms
+from django.contrib import admin, messages
+
+from django.utils.translation import gettext_lazy as _
+
+from .models import Product, Transaction
+
+
+class ProductAdmin(admin.ModelAdmin):
+    """Admin handler for Products"""
+
+    list_display = ("id", "price", "name", "desc", "category", "associated_tournament")
+    search_fields = ["id", "price", "name"]
+
+
+admin.site.register(Product, ProductAdmin)
+
+
+@admin.action(description=_("Rembourser la transaction"))
+def reimburse_transactions(modeladmin, request, queryset):
+    """Reimburse all selected actions"""
+    for transaction in queryset:
+        (is_err, msg) = transaction.refund(request.user.username)
+        if is_err:
+            modeladmin.message_user(request, _("Erreur: %s") % msg, messages.ERROR)
```

Comment:
f-string is more a more modern syntax, and as it used across the code base, we may switch to f-strings

Reply:
[The documentation](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#standard-translation) still shows `%` but I think those should be equivalent